python/pyspark/sql/column.py

```diff
@@ -364,6 +364,22 @@ def isin(self, *cols):
     isNull = ignore_unicode_prefix(_unary_op("isNull", _isNull_doc))
     isNotNull = ignore_unicode_prefix(_unary_op("isNotNull", _isNotNull_doc))
 
+    def isNotDistinctFrom(self, col):
```

Comment:
Actually, I like this name. `IS [NOT] DISTINCT FROM` is part of ANSI SQL. We should add alias in Spark SQL too.